docs/database/db_schema-main.sql

```diff
@@ -59,6 +58,21 @@ CREATE TABLE `certificateStatus` (
   PRIMARY KEY (`serial`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `challenges` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `authorizationID` char(43) NOT NULL,
+  `LockCol` bigint(20) DEFAULT NULL,
+  `type` varchar(255) NOT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `error` varchar(255) DEFAULT NULL,
+  `validated` datetime DEFAULT NULL,
+  `uri` varchar(255) DEFAULT NULL,
+  `token` char(43) NOT NULL,
+  `tls` tinyint(1) DEFAULT NULL,
+  `validation` mediumblob,
```

Comment:
Please still fix the authorizationID. Being consistent and slightly too large than we need is better than inconsistent and screwing this up later.

I'm concerned about these stringly typed fields, still. What's our plan? What happens when we add a status or type that is larger than 10 characters?